packages/formation-react/src/components/MegaMenu/MegaMenu.jsx

```diff
@@ -159,7 +159,7 @@ export default class MegaMenu extends React.Component {
                         {
                           item.title === currentDropdown && item.menuSections && <ul aria-label={item.title}>
                             {
-                              item.menuSections.constructor.name === 'Array' ? item.menuSections.map((section, j) => {
+                              Array.isArray(item.menuSections) ? item.menuSections.map((section, j) => {
```

Comment:
this check fails on IE 11. 

sometimes `menusections` is an array. sometimes it's an object. its rendering behavior probably should be based on something more explicit than its datatype. 

![Screen Shot 2019-03-11 at 15 16 52](https://user-images.githubusercontent.com/4998130/54158703-bd715f00-4410-11e9-842f-bb16811641c3.png)
---
![Screen Shot 2019-03-11 at 15 16 42](https://user-images.githubusercontent.com/4998130/54158712-c2cea980-4410-11e9-9d12-ba0d01cd2a49.png)

Reply:
Looks like this could have been broken for 7 months 😱